cypress/tests/overview/settings-cluster-general.cy.ts

```diff
@@ -0,0 +1,107 @@
+import { SECOND } from '../../utils/const/index';
+import { tab } from '../../views/tab';
+
+describe('Test Cluster General settings', () => {
+  before(() => {
+    cy.login();
+    cy.visit('');
+    cy.wait(10000);
```

Comment:
IMHO the `wait()` here (and in different places too) gives a false sense of security - we should embed the timeout in `cy.get()` calls. 
i.e. a good example is ` switchPerspective()` where wait up to 5 minutes (but will proceed as soon as possible)
https://github.com/kubevirt-ui/kubevirt-plugin/blob/67d8f730d8c3c49347209683c8fa8667fb172a09/cypress/views/perspective.ts#L19
not so good example is a `wait()` below (line 20)  which should be embedded (and increased) in the next `get()` call i.e.
``
 cy.get(toggle, { timeout: 5 * MINUTE }).click()`
```

Reply:
The timeout outside of `cy.get` is what I learned from the test execution, there is flakiness that the page will refresh/flash when try to open a dropdown menu and it brokes a follow-up click.

We have lots of similar case in D/S tests and it's necessary.